docs/md/modules-directory-usage.md

```diff
@@ -7,19 +7,19 @@ for the specific pipeline for more information).
 If you have a set of files uploaded to a GCS bucket and you'd like to start
 a workflow for each one, you can do that via shell scripting.
 
-!!! warning "Note"
-    You will likely run into performance issues with WFL if you try to start
-    hundreds or thousands of workflows in a single request to WFL. You'll need
-    to split up the workflows into multiple workloads, tips for that are
-    [here](#other-notes).
+??? question "Wondering about performance?"
```

Comment:
I think this bit can be removed - the user shouldn't really care how we're calling cromwell.

Reply:
Totally makes sense, will remove!